docs/azmcp-commands.md

```diff
@@ -354,6 +354,75 @@ azmcp postgres server setparam --subscription <subscription> \
                                --value <value>
 ```
 
+### Azure Database for MySQL Operations
+
+#### Database commands
+
+```bash
+# List all databases in a MySQL server
+azmcp mysql database list --subscription <subscription> \
+                          --resource-group <resource-group> \
+                          --user-name <user> \
+                          --server <server>
+
+# Execute a query on a MySQL database
+azmcp mysql database query --subscription <subscription> \
+                           --resource-group <resource-group> \
+                           --user-name <user> \
+                           --server <server> \
+                           --database <database> \
+                           --query <query>
```

Comment:
If users are allowed to input queries directly, is there a risk of SQL injection?

Reply:
yes, same risk as other sql tools. One recommendation would be to configure database users with the minimum necessary permissions.

Reply:
Can we validate the query before execution?

Reply:
We can analyze the SQL query into an AST (abstract syntax tree) before execution. If the AST contains any InsertStatement, UpdateStatement, DeleteStatement, or DDL nodes (Create, Alter, Drop, etc.), you reject the query. Ensure that only SelectStatement nodes are present. 

For SQL Server : https://github.com/microsoft/SqlScriptDOM
Need to find equivalents for Postgres and mysql

Reply:
Will create a separate issue+PR to handle this.

Reply:
No. I still need to think through this properly. Will handle in separate PR.